specs/fulu/validator.md

```diff
@@ -43,6 +49,55 @@ document and used throughout.
 | - | - | - |
 | `VALIDATOR_CUSTODY_REQUIREMENT` | `8` | Minimum number of custody groups an honest node with validators attached custodies and serves samples from |
 | `BALANCE_PER_ADDITIONAL_CUSTODY_GROUP` | `Gwei(32 * 10**9)` | Balance increment corresponding to one additional group to custody |
+| `MAX_CELLS_PER_EXT_BLOB` | `1024` | Corresponds to a cell with 8 field elements per blob |
+
+## Helpers
+
+### `BlobsBundle`
+
+*[Modified in Fulu:EIP7594]*
+
+The `BlobsBundle` object is modified to include cell KZG proofs instead of blob KZG proofs.
+
+```python
+@dataclass
+class BlobsBundle(object):
+    commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
+    # [Modified in Fulu:EIP7594]
+    proofs: List[KZGProof, MAX_CELLS_PER_EXT_BLOB * MAX_BLOB_COMMITMENTS_PER_BLOCK]
```

Comment:
```suggestion
    proofs: List[KZGProof, FIELD_ELEMENTS_PER_BLOB * MAX_BLOB_COMMITMENTS_PER_BLOCK]
```
`FIELD_ELEMENTS_PER_BLOB` is an existing constant and the theoretical maximum. Since this SSZ type is only serialized we just need to choose a value that's higher than the expected length of the list.

Reply:
Okay I understand. I like this idea. I've decided to use `FIELD_ELEMENTS_PER_EXT_BLOB` instead though. This would accommodate cells with 1 field element per cell; literally the maximum. You good with this?